src/Polished/Color/Polished_Color_Mix.re

```diff
@@ -0,0 +1,96 @@
+open Polished_Types;
+open Polished_Color_Utils;
+
+/**
+ * Returns [color] and [baseColor], mixed together and weighted by [weight]
+ */
+let mix = (color: color, baseColor: color, weight: Percent.t): color => {
```

Comment:
Wondering if it'd be better to use labelled arguments here like this:
`let mix = (color: color, ~base: color, ~weight: Percent.t): color =>`
It would make it very clear which one of the colors is base and which is the "main" when you read the code, see:
`let mixed = myColor->mix(~base=baseColor, ~weight=someWeight)`
What do you think? I'm not sure. Is it important to not mix up base and main color by accident? Because this would slightly avoid with that. But if the order doesn't really matter, then this might be unnecessarily complex.

Reply:
Hmm, maybe not after all. I was thinking about [Readable](https://github.com/Rohea/re-polished/blob/master/src/Polished/Color/Polished_Color_Readable.re#L14-L21) function but it has 3 input colors this one has only two. Also all of the variables are required here.